### PR TITLE
US771133/US868107: Correctly return responses from new endpoints

### PR DESCRIPTION
**Pull request by jonny-mccoubrey**
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=771133
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=868107

### CODE REVIEW

File: job-service/src/main/java/com/hpe/caf/services/job/api/generated/JobsApiServiceImpl.java
Comment (michael-bryson) [inline job-service/src/main/java/com/hpe/caf/services/job/api/generated/JobsApiServiceImpl.java]:
```suggestion
        return Response.ok().entity(String.format("Successfully canceled %s jobs", successfulCancellations)).build();
```
This should be US spelling unfortunately

**Review by andyreidz — CHANGES_REQUESTED** [review]
Apologies for not thinking about this in the first review, let's return the number of canceled or deleted jobs instead of a string, that way it will be easier for a caller to validate the response.

**Comment by jonny-mccoubrey** [comment]
This PR was initially correcting the MIME type that the two new endpoints were returning.
Initially we were returning a String response so MIME type `text/plain` was being used.

However, when we decided to return Long response instead, `text/plain` caused an error. For the Long response, we choose MIME type `application/json` and it works as expected.

**Comment by jonny-mccoubrey** [comment]
NOTE: Have requested review but will wait for CI build of final commit to pass before merging

**Review by andyreidz — APPROVED** [review]

**Pull request merged**
